doc/project_governance/technical_committee.md

```diff
@@ -28,5 +28,5 @@ The OpenTitan Technical Committee membership is:
 * Neeraj Upasani
 * Nir Tasher
 * Arnon Sharlin
-* Sivakumar
 * Tim Trippel
+* Miles Dai
```

Comment:
Somewhat nitty, but I think the members are currently sorted by last name, so I think we should keep it that way